editor/src/core/model/element-metadata-utils.ts

```diff
@@ -410,6 +410,12 @@ export const MetadataUtils = {
   isPositionAbsolute(instance: ElementInstanceMetadata | null): boolean {
     return instance?.specialSizeMeasurements.position === 'absolute'
   },
+  isFlowElement(instance: ElementInstanceMetadata | null): boolean {
+    return (
+      instance?.specialSizeMeasurements.immediateParentProvidesLayout === false &&
```

Comment:
hmm do we need to check for `instance?.specialSizeMeasurements.immediateParentProvidesLayout === false`? why not `instance?.specialSizeMeasurements.parentLayoutSystem === 'flow'` ?

Reply:
fixed! I had some confusion about the specialSizeMeasurements